VisualizationBase/src/Scene.cpp

```diff
@@ -606,6 +606,7 @@ void Scene::removeAllOnZoomHandlers()
 int Scene::addOnZoomHandler(OnZoomHandler onZoomHandler, OnZoomHandlerRemove onZoomHandlerRemove)
 {
 	onZoomHandlers_.insert(addOnZoomHandlerId_, {onZoomHandler, onZoomHandlerRemove});
+	onZoomHandler(mainViewScalingFactor_);
```

Comment:
There is one place where you call this method, and I think it's more correct for your to call the handler there, once you've added it. The method here should not change.